app/assets/stylesheets/application.scss

```diff
@@ -12,9 +12,11 @@ body {
   padding-right: 5%;
 }
 
+/* stylelint-disable */
```

Comment:
Tev nevajadzeja disablot visus rules priekš šis rindas, tikai to ar kebab-case. Tāpēc kā var gaditites, kā bloks tiks papildināts un tur jau paradisies pavisam cita problēma.

Pamegini, lūdzu, vel izmantot šo "stylelint-disable-next-line"
Vnk būs daudz lasamak un mums nebus vesels bloks, bet tikai viena rindiņa. Palasi par to dokos, kurus tev metu

Reply:
Pamegini velreiz